sdk/src/client/bindings/python/iota_client/types/address.py

```diff
@@ -0,0 +1,67 @@
+from enum import Enum
+
+class AddressType(Enum):
+    ED25519 = 0
+    ALIAS = 8
+    NFT = 16
+
+class Address():
+    def __init__(self, type, address_or_id):
+        """Initialize an Address
+        
+        Parameters
+        ----------
+        type : AddressType
+            The type of the Address
+        address_or_id : string
+            The address to use. Can either be an hex encoded ED25519 address or NFT/Alias id
+        """
+        self.type = type
+        self.address_or_id = address_or_id
+
+    def as_dict(self):
+        config = dict(self.__dict__)
+
+        config['type'] = config['type'].value
+        
+        if self.type == AddressType.ED25519:
+            config['pubKeyHash'] = config.pop('address_or_id')
+        elif self.type == AddressType.ALIAS:
+            config['alias_id'] = config.pop('address_or_id')
```

Comment:
There is a mix of snake_case and camelCase, is this correct?

Reply:
No it's not. Good catch. Looking at my old jsonschema it should be `aliasId` 🤔